todo/src/components/Todo.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { addTodo, deleteTodo, toggleTodo, clearCompleted } from '../actions';
+
+
+class Todo extends React.Component {
+    state = {
+        newTodo: ''
+    };
+     
+    handleChanges = e => {
+        this.setState({ newTodo: e.target.value});
+    };
+
+    addTodo = e => {
+        e.preventDefault();
+        this.props.addTodo(this.state.newTodo);
+        this.setState({ newTodo: ''})
+    };
+
+    deleteTodo = id => {
+        this.props.deleteTodo(id);
+    };
+
+    toggleTodo = id => {
+        this.props.toggleTodo(id);
+    };
+
+    clearCompleted = id => {
```

Comment:
don't need to take an `id` here since you're not using that information in the reducer